assets/pages/subscriptions-landing/components/featuredProducts.jsx

```diff
@@ -0,0 +1,87 @@
+// @flow
+
+// ----- Imports ----- //
+import React, { type Node } from 'react';
+
+import { type SubsUrls } from 'helpers/externalLinks';
+import { getQueryParameter } from 'helpers/url';
+import { type SubscriptionProduct } from 'helpers/subscriptions';
+
+import GridPicture from 'components/gridPicture/gridPicture';
+
+
+// ----- Types ----- //
+type Product = {|
+  name: SubscriptionProduct,
+  headingText: string,
+  bodyText: string,
+  link: string,
+  image: Node,
+|}
+
+
+// ----- Exports ----- //
+const dpImage = (
+  <GridPicture
+    sources={[
+      {
+        gridId: 'digitalPackFlashSaleMobile',
+        srcSizes: [140, 500, 717],
+        imgType: 'png',
+        sizes: '90vw',
+        media: '(max-width: 739px)',
+      },
+      {
+        gridId: 'digitalPackFlashSaleDesktop',
+        srcSizes: [140, 500, 1000, 1388],
+        imgType: 'png',
+        sizes: '(min-width: 1300px) 750px, (min-width: 1140px) 700px, (min-width: 980px) 600px, (min-width: 740px) 60vw',
+        media: '(min-width: 740px)',
+      },
+    ]}
+    fallback="digitalPackFlashSaleDesktop"
+    fallbackSize={500}
+    altText="ad-free, live and discover, and the daily edition"
+    fallbackImgType="png"
+  />);
+
+
+const getProducts = (subsLinks: SubsUrls): {DigitalPack: Product, Paper: Product, GuardianWeekly: Product} => ({
+  DigitalPack: {
+    name: 'DigitalPack',
+    headingText: 'Digital Pack',
+    bodyText: 'Read the Guardian ad-free on all devices, plus get all the benefits of the Premium App and Daily Edition iPad app of the UK newspaper',
```

Comment:
I think we might need to localise this copy. Usually the UK version just says 'the newspaper' and everywhere else says 'the UK newspaper'.

Reply:
do we have any libraries in place for localising strings?

Reply:
[This](https://github.com/guardian/support-frontend/blob/master/assets/helpers/contributions.js#L134) is a pattern we use in a few places.